linux/Dockerfile

```diff
@@ -2,17 +2,8 @@ FROM ubuntu:focal
 
 # Install the packages contained in `packages.txt`
 COPY packages.txt /opt/crates-build-env/packages.txt
-RUN apt-get update && cat /opt/crates-build-env/packages.txt | DEBIAN_FRONTEND=noninteractive xargs apt-get install -y
-
-# Fix the "sudo: setrlimit(RLIMIT_CORE): Operation not permitted" error when
-# starting up the container with `MAP_USER_ID`. Upstream issue:
-#
-#    https://github.com/sudo-project/sudo/issues/42
-#
-RUN echo "Set disable_coredump false" >> /etc/sudo.conf
-
-# Setup the entrypoint
-COPY entrypoint.sh /opt/crates-build-env/entrypoint.sh
-ENTRYPOINT ["/opt/crates-build-env/entrypoint.sh"]
+RUN apt-get update && \
+    cat /opt/crates-build-env/packages.txt | DEBIAN_FRONTEND=noninteractive xargs apt-get install -y && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
Is there a way to test this without deploying a new image to production?

Reply:
The easiest way to test this is to build the micro image locally and run docs.rs or Crater with it.

The `rm -rf` does not break anything, and removes around 30MB from the resulting image (less prominent on the full image but noticeable on the micro one). It just deletes the cached APT packages list, and running `apt-get update` again brings it back in if something actually needs it.

Reply:
Thanks, that worked :)